include/kodlab_mjbots_sdk/mjbots_control_loop.h

```diff
@@ -41,15 +41,15 @@ struct ControlLoopOptions {
 
 /*!
  * @brief mjbots_control_loop class is an parent class to be used to create a control loop. It supports 1 controller and
- *        logging. The child class must implement CalcTorques and PrepareLog (if logging). The robot data is stored in
+ *        logging. The child class must implement Update and PrepareLog (if logging). The robot data is stored in
  *        the robot object. The behavior runs in its own thread. To Start the thread Run Start()
  * @tparam LogClass[optional] data type for logging
  * @tparam InputClass[optional] class for input data 
  * @tparam RobotClass[optional] RobotInterfaceDerived class that contains state and control calculations 
```

Comment:
Comment needs to be updated

Reply:
Resolved in 8962188d140e046fbc1ade97323bd3b6774e975c.